partitura/musicanalysis/note_features.py

```diff
@@ -1073,6 +1073,44 @@ def metrical_strength_feature(na, part, **kwargs):
 
     return W, names
 
+def measure_feature(na, part, **kwargs):
+    """Measure feature
+
+    This feature encodes the measure each note is in.
+
+    """
+    notes = part.notes_tied if not np.all(na["pitch"] == 0) else part.rests
+    bm = part.beat_map
+    
+    global_start = bm(part.first_point.t)
+    global_end = bm(part.last_point.t)
+    global_number = 0 # default global measure number
+
+    names = [
+        "measure_number",
+        "measure_start_beat",
+        "measure_end_beat",
+    ]
+    W = np.zeros((len(notes), 3))
+
+    for i, n in enumerate(notes):
+        measure = next(n.start.iter_prev(score.Measure, eq=True), None)
+
+        if measure:
+            start = bm(measure.start.t)
+            end = bm(measure.end.t)
```

Comment:
user could add measures without start or end point. in our Measure refactor #311 and discussion we said that for built-in parsers, a measure must always have a start and an end. I think the usage of Measures without start and end point can result in an error.

Reply:
I am slightly worried that this could have the same effect as the Tuplet assertion I implemented in the last release, where the construct is too strict and often results into errors. However, this usecase would be much less common than import. Maybe we could add an explicating assertion for that case, or not. I leave it up to you. 

You can resolve the comment and then we can merge